common-rest/src/main/java/uk/ac/ebi/uniprot/api/rest/controller/BasicSearchController.java

```diff
@@ -113,4 +126,18 @@ protected BasicSearchController(ApplicationEventPublisher eventPublisher,
 
     protected abstract String getEntityId(T entity);
 
+    protected abstract Optional<String> getEntityRedirectId(T entity);
+
```

Comment:
can remove extra new line

Reply:
Line removed... I also removed other extra lines in this class